test/unit/tracks/text-track-settings.test.js

```diff
@@ -9,6 +9,16 @@ const tracks = [{
   label: 'test'
 }];
 
+const defaultSettings = {
```

Comment:
@gkatsev I needed to do some more research on this; I was wrong about "basic Text Tracks do not specify a foreground or background color, and nor does vtt.js when it parses them". 

The idea of `---` or `default` in the Captions Settings Dialog is somewhat bogus, since if a Text Track or cue doesn't specify its own styling, then the `default` is [buried in vtt.js](https://github.com/mozilla/vtt.js/blob/master/lib/vtt.js#L589-591), and is entirely arbitrary; neither the WebVTT standard nor anything from the FCC specifies a "default" setting for caption display if none is specified by the captions track. In my mind, the vtt.js default is equivalent to the [`Need Text` in video.js' components](https://github.com/videojs/video.js/blob/master/src/js/clickable-component.js#L95) - it's there in case nothing else is specified.

Which takes me back to my point that there shouldn't be a default in drop downs the video.js Captions Settings Dialog; there should be a default *set of settings* for color, opacity/transparency and font/outline/etc. which can match the existing behavior of vtt.js for consistency, and a new setting which should be an option of whether this *overrides* any color specified in the Text Track or not.

Whether the settings are saved by video.js and then restored (the "persistence option") is a separate feature from all of this.